core/templates/dev/head/profile/signup.html

```diff
@@ -156,6 +156,7 @@ <h2 class="oppia-signup-page-title">Complete Your Registration</h2>
       <button class="btn btn-default" ng-click="close()">Close</button>
     </div>
   </script>
+  {% include 'footer.html' %}
```

Comment:
Actually let's drop the inclusion on this page, since the page is quite action-oriented.